utils/bin/lvim.ps1

```diff
@@ -0,0 +1,8 @@
+$env:HOME = "$HOME"
+$env:USER = "$HOME"
+$env:SHELL = "cmd"
+$env:TERMINAL = "$env:SHELL"
+$env:VIMRUNTIME = "$(scoop prefix neovim-nightly)/share/nvim/runtime";
```

Comment:
I'm not sure everyone is installing with scoop necessarily, so it should not be a default value.

Reply:
It's just a template for the reference